types/chai-subset-in-order/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for chai-subset-in-order 3.1
+// Project: https://github.com/oprogramador/chai-subset-in-order
+// Definitions by: Mario Ramundo <https://github.com/ramundomario>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
```

Comment:
```suggestion

/// <reference types="chai" />

```
(for `Cahi.ChaiPlugin` usage and resolution outside of DT repo)

Reply:
Sure! Thanks a lot for the review 🙏